backend/test-app.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+import requests
+import numpy as np
+from app import *
+
+# parse(forecast)
+
+# def testParse(url):
+    # if parse(url) == ['Rest Of Today', 'Mostly cloudy late in the morning then becoming mostly sunny. Highs in the mid 30s. Northwest winds 5 to 10 mph.', 'Tonight', 'Mostly clear. Areas of fog after midnight. Lows around 16. Northwest winds up to 5 mph.']: # Output
+        # print("🎉")
+    # else:
+        # print("😭")
+    # if parse(url) != [' ']: # Empty
+        # print("🎉")
+    # else:
+        # print("😭")
+    # return 
+
+# url = 'https://api.weather.gov/zones/forecast/MNZ060/forecast' 
```

Comment:
It's okay to test the parse method using this url, but for testing the API we should be making a request to the API (`localhost:3001` when running).